ui/src/view/button.rs

```diff
@@ -28,6 +46,26 @@ where
     T: Clone,
 {
     pub fn view(self) -> Node<T> {
-        button![self.label, ev(Ev::Click, |_| self.on_click)]
+        let tag = match self.on_click {
+            Handler(_) => "button",
+            Route(_) => "a",
+        };
```

Comment:
If the button's click handling is a `Route` variant we need to be an `a` element, otherwise we need it to be a `button`